tests/baselines/reference/docker/xterm.js.log

```diff
@@ -1,9 +1,30 @@
-Exit Code: 0
+Exit Code: 2
 Standard output:
 
 > xterm@X.X.X build /xtermjs
 > tsc -b ./tsconfig.all.json
+node_modules/@types/ws/index.d.ts(44,39): error TS2694: Namespace '"url"' has no exported member 'URL'.
```

Comment:
@sandersn I'm going to merge this update, but we should probably check that this is an intentional break at DT. It's looking like this is an incompatibility between newer `@types/node` (?) and `@types/ws`.